harness-maven-plugin/src/main/java/com/sun/xml/ws/test/WSTestMojo.java

```diff
@@ -599,7 +599,7 @@ private String getHarnessClassPath() throws MojoExecutionException {
             throw new MojoExecutionException("Couldn't download artifact: " + e.getMessage(), e);
         }
         return arr.getArtifacts();
-    }
+    }*/
```

Comment:
why this has been commented out?

Reply:
for this method we have to use maven-artifact 2.2.1, it's not accepted by PLS, but the latest maven-artifact 3.5.2 is not compatible with old version.  I commented this method and tested  jaxws-ri/tests and wsit/tests did not find any problem.
or can you provide advices to replace this method?

Reply:
The best would be to update this to component RepositorySystem - try to take a look at glassfish build plugin which is using that approach to resolve dependencies

Reply:
See https://github.com/javaee/glassfishbuild-maven-plugin/blob/master/src/main/java/org/glassfish/build/FeatureSetsDependenciesMojo.java. PLS should be valid since this should be included with the GlassFish donation (to be verified)